src/hsmm_mvpy/simulations.py

```diff
@@ -60,8 +60,7 @@ def event_shape(event_width, event_width_samples, steps):
     template = template/np.sum(template**2)#Weight normalized
     return template
 
-
-def simulate(sources, n_trials, n_jobs, file, data_type='eeg', n_subj=1, path='./', overwrite=False, verbose=False, noise=True, times=None, location=1, seed=None): 
+def simulate(sources, n_trials, n_jobs, file, data_type='eeg', n_subj=1, path='./', overwrite=False, verbose=False, noise=True, times=None, seed=None, sfreq=None): 
```

Comment:
On the sfreq it probably needs a default or a catch for is None in line 135

Reply:
I would say we can just go for a default at 100Hz

Reply:
good point, yes 100hz would be good here!